deploy/service.yaml

```diff
@@ -180,7 +225,7 @@ spec:
         - name: distributor
           image: keptn/distributor:0.9.2
```

Comment:
Version `0.10.0` of the distributor is already released. You could use it here :)